runtime/polkadot/src/lib.rs

```diff
@@ -104,34 +103,27 @@ pub fn native_version() -> NativeVersion {
 	}
 }
 
-/// Avoid processing transactions that are anything except staking and claims.
-///
-/// RELEASE: This is only relevant for the initial PoA run-in period and may be removed
-/// from the release runtime.
-#[derive(Default, Encode, Decode, Clone, Eq, PartialEq, RuntimeDebug)]
-pub struct OnlyStakingAndClaims;
-impl SignedExtension for OnlyStakingAndClaims {
-	const IDENTIFIER: &'static str = "OnlyStakingAndClaims";
-	type AccountId = AccountId;
-	type Call = Call;
-	type AdditionalSigned = ();
-	type Pre = ();
-
-	fn additional_signed(&self) -> sp_std::result::Result<(), TransactionValidityError> { Ok(()) }
-
-	fn validate(
-		&self, _:
-		&Self::AccountId,
-		call: &Self::Call,
-		_: &DispatchInfoOf<Self::Call>,
-		_: usize
-	)
-		-> TransactionValidity
-	{
+pub struct IsCallable;
+impl Filter<Call> for IsCallable {
+	fn filter(call: &Call) -> bool {
 		match call {
-			Call::Slots(_) | Call::Registrar(_)
-				=> Err(InvalidTransaction::Custom(ValidityError::NoPermission.into()).into()),
-			_ => Ok(Default::default()),
+			// Governance stuff
+			Call::Democracy(_) | Call::Council(_) | Call::TechnicalCommittee(_) |
+			Call::ElectionsPhragmen(_) | Call::TechnicalMembership(_) | Call::Treasury(_) |
+			// Parachains stuff
+			Call::Parachains(_) | Call::Attestations(_) | Call::Slots(_) | Call::Registrar(_) |
+			// Balances and Utility (which can be used to call other stuff)
+			Call::Balances(_) | Call::Utility(_) | Call::Indices(_) =>
+				false,
+
+			// These modules are all allowed to be called by transactions:
+			Call::System(_) | Call::Scheduler(_) |
+			Call::Babe(_) | Call::Timestamp(_) |
+			Call::Authorship(_) | Call::Staking(_) | Call::Offences(_) |
+			Call::Session(_) | Call::FinalityTracker(_) | Call::Grandpa(_) | Call::ImOnline(_) |
+			Call::AuthorityDiscovery(_) |
+			Call::Claims(_) | Call::Vesting(_) | Call::Sudo(_) =>
```

Comment:
Wouldn't this allow `vested_transfer` to be called? https://github.com/paritytech/substrate/blob/master/frame/vesting/src/lib.rs#L252